returnn/datasets/distrib_files.py

```diff
@@ -6,7 +6,7 @@
 
 from __future__ import annotations
 
-from typing import Optional, Any, Dict, Tuple, List, Sequence, Callable, Union
+from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
```

Comment:
Both the old order but also the new order are very inconsistent to how we usually order these in RETURNN.
We usually start with generic ones (`Optional`, `Union`, `Any`), and it gets more specific to the right, i.e. things like `List`, `Tuple`, `Dict` come last. `Sequence` before those. `Callable`, `TypeVar` etc usually also before.

Reply:
```suggestion
from typing import Union, Optional, Any, Callable, Sequence, Tuple, List, Dict
```

Reply:
I would say, let's move this to a separate issue/discussion and not discuss here. So here in this PR, I would either not change the import order at all now, or make it consistent to remaining RETURNN code.